packages/vue/src/pages/crm-contact-list/crm-contact-list.vue

```diff
@@ -0,0 +1,255 @@
+<template>
+  <div class="view-wrapper">
+    <dx-data-grid ref="dataGrid"
+                  :data-source="gridData"
+                  height="100%"
+                  class="grid"
+                  @row-click="rowClick">
+
+      <!-- Options -->
```

Comment:
Why did you add this comment?

Reply:
I like to add such comments for many lines templates, i think it's more readable